job_spec/AVO.yml

```
@@ -0,0 +1,109 @@
+AVO:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default:  '""'
+      api_schema:
+        type: array
+        minItems: 2
+        maxItems: 2
+        example:
+          - S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8
+          - S1A_IW_SLC__1SSV_20150504T120217_20150504T120229_005771_00769E_EF9A
+        items:
+          description: The name of the Sentinel-1 SLC granule to process
+          type: string
+          pattern: "^S1[AB]_IW_SLC__1S[SD][VH]"
+          minLength: 67
+          maxLength: 67
+          example: S1A_IW_SLC__1SSV_20150621T120220_20150621T120232_006471_008934_72D8
+    bucket_prefix:
+      default:  '""'
+    include_look_vectors:
+      api_schema:
+        description: Include the look vector theta and phi files in the product package
+        default: false
+        type: boolean
+    include_los_displacement:
+      api_schema:
+        description: Include a GeoTIFF in the product package containing displacement values along the Line-Of-Sight (LOS)
+        default: false
+        type: boolean
+    include_displacement_maps:
+      api_schema:
+        description: Include displacement maps (line-of-sight and vertical) in the product package
+        default: false
+        type: boolean
+    include_inc_map:
+      api_schema:
+        description: Include the incidence angle map(s) in the product package
+        default: false
+        type: boolean
+    include_dem:
+      api_schema:
+        description: Include the DEM file in the product package
+        default: false
+        type: boolean
+    include_wrapped_phase:
+      api_schema:
+        description: Include the wrapped phase GeoTIFF in the product package
+        default: false
+        type: boolean
+    apply_water_mask:
+      api_schema:
+        description: Sets pixels over coastal and large inland waterbodies as invalid for phase unwrapping.
+        default: false
+        type: boolean
+    looks:
+      api_schema:
+        description: Number of looks to take in range and azimuth
+        type: string
+        default: 20x4
+        enum:
+          - 20x4
+          - 10x2
+    phase_filter_parameter:
+      api_schema:
+        description: parameter for the adaptive phase filter.
+        type: float
```

```suggestion
        type: number
        minimum: 0.0
        maximum: 1.0
```
https://swagger.io/docs/specification/data-models/data-types/